lib/environment.sh

```diff
@@ -34,6 +34,12 @@ list_node_config() {
     echo "npm scripts will see NODE_ENV=production (not '${NODE_ENV}')"
     echo "https://docs.npmjs.com/misc/config#production"
   fi
+
+  if [ "$NPM_CONFIG_PRODUCTION" == "true" ]; then
+    mcount "npm-config-production-true"
+  elif [ "$NPM_CONFIG_PRODUCTION" == "false" ]; then
+    mcount "npm-config-production-false"
+  fi
```

Comment:
no `else`?

Reply:
I can get the `else` by total # of builds. I just want to check the % that have it set to each value.